src/dmd/dinterpret.d

```diff
@@ -47,6 +47,28 @@ import dmd.tokens;
 import dmd.utf;
 import dmd.visitor;
 
+
+void incUsageCtfe(InterState* istate, const ref Loc loc)
+{
+    if (global.params.cov && istate)
+    {
+        auto line = loc.linnum;
+        auto mod = istate.fd.getModule();
+        if (mod.ctfe_cov is null)
+        {
+            mod.ctfe_cov = [line:0];
+        }
+
+        if (auto count = line in mod.ctfe_cov)
+        {
+            (*count)++;
+        }
+        else
+        {
+            mod.ctfe_cov[line] = 1;
+        }
```

Comment:
You can replace all of this code by `++mod.ctfe_cov[line]`, as built-in AAs automatically initialize values on mutation.

Reply:
they do?
I think that wasn't the case in D1 :)

Reply:
Thanks @tgehr